release/gff/gff.ti-ethi.gff_tigrinya/gff.ti-ethi.gff_tigrinya.model_info

```diff
@@ -0,0 +1,7 @@
+{
+    "license": "mit",
+    "languages": [
+        "ti-ET", "ti-ER"

```

Comment:
Issue keymanapp/keyman#1285 is still open

Should this list include `ti-Ethi-ET`, `ti-Ethi-ER`?

Reply:
Uncertain on this. There is no alternative script for writing Tigrinya, so Ethi is always the default.  I've kept out Ethi for now, this would be consistent with the Amharic package.